workshops/avd-lab-guide.md

```diff
@@ -823,4 +823,201 @@ git branch -D add-leafs
+
+Luckily, AVD has an inband management data model, which is very lean and easy to deploy.  The full data model can be reviewed right on the **[AVD documentation site](https://avd.arista.com/4.5/roles/eos_designs/docs/input-variables.html#node-type-inband-management)**.
+
+This data model is deployed in the **Fabric** vars file, either under the node type defaults, or under specific nodes.  While there are many options to configure, for this lab, we will just be focusing on the following:
+
+``` yaml
+<node_type_keys.key>:
+
+  # Define variables for all nodes of this type.
+  defaults:
+
+    # VLAN number used for inband management on L2 switches (switches using port-channel trunks as uplinks).
+    # When using 'inband_mgmt_subnet' the VLAN and SVIs will be created automatically on this switch as well as all 'uplink_switches'.
+    # When using 'inband_mgmt_ip' the VLAN and SVI will only be created on this device and added to uplink trunk. The VLAN and SVI on the parent switches must be created using network services data models.
+    inband_mgmt_vlan: <int; default=4092>
+
+    # Optional IP subnet assigned to inband management SVIs on L2 switches (switches using port-channels as uplinks).
+    # SVI IP address will be assigned as follows:
+    # virtual-router: <subnet> + 1
+    # l3leaf A      : <subnet> + 2 (same IP on all l3leaf A)
+    # l3leaf B      : <subnet> + 3 (same IP on all l3leaf B)
+    # l2leafs       : <subnet> + 3 + <l2leaf id>
+    # GW on l2leafs : <subnet> + 1
+    # Assign range larger than total l2leafs + 5
+
+    # This setting is applicable to L2 switches (switches using port-channel trunks as uplinks).
+    inband_mgmt_subnet: <str>
+```
+
+With just these two key value pairs, AVD will automatically create the inband management VLAN, its associated SVIs on the spines, the vARP virtual IP for the subnet gateway, the VLAN and its SVIs on the leafs, and the static route.
+
+Lets get to work creating our inband management network.
+
+#### **Branch Time**
+
+Following our development best practices, let's create a new branch for our work. We'll call this branch **inband-mgmt**.
+
+```bash
+git switch -c inband-mgmt
+```
+
+Now that we are on our new working branch, lets add our data model and render our configs.
+
+#### **Data Model Creation**
+
+Next, let's add our inband management data model into `sites/site_1/group_vars/SITE1_FABRIC.yml`.
+
+This will be the actual data model we will drop into our `SITE1_FABRIC` vars file:
+
+```yaml
+inband_mgmt_vlan: 3000
+inband_mgmt_subnet: 10.255.255.0/24
+```
+
+The data model should be nested in the vars file like this:
+
+```yaml
+# Leaf Switches
+leaf:
+  defaults:
+    platform: cEOS
+    < insert inband management keys here >
```

Comment:
I would suggest adding the needed keys here and eliminate the the 2 lines above.

Reply:
> Have you tested this data models work in the ATD lab? How are you showing them how to connect to the new in-band mgmt address or you just showing that it gets created?

Just how it gets created.  I guess you could connect to it from the linux host but not sure its worth it.

Reply:
Has it been tested in the ATD Lab environment?

Reply:
> Has it been tested in the ATD Lab environment?

No but I have that exact data model running in a production environment, and its not new.  Dont see why it wouldnt work.

Reply:
The ATD Labs have been updated to AVD 4.5 and ansible-core 2.15.5, so it would be worth testing your data model before we merge this.